### PR TITLE
Rebuild for Python 3.11

### PR DESCRIPTION
**Pull request by xhochy**
Fixes #95 

FYI @ivergara 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xhochy** [comment]
Probably depends on https://github.com/conda-forge/watchfiles-feedstock/pull/18

File: conda-forge.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-bot: {automerge: true}
+# Don't automerge, uvicorn-standard dependencies regularly change and need to edited manually.
+# bot: {automerge: true}
```
Comment (ocefpaf) [inline conda-forge.yml:2]: You can activate grayskull hinting here to have those updates in the bot PR. I'd still keep the automerge off though.

Comment (xhochy) [reply conda-forge.yml:2]: grayskull doesn't hint on extra requirements. That's what we would need here.

Comment (ocefpaf) [reply conda-forge.yml:2]: Not sure what you mean by extra. Extra in like, optional but I the python package metadata? Or in external non-python does? The former should work.

**Comment by ivergara** [comment]
@xhochy is this mergeable now?

**Comment by xhochy** [comment]
Yes!x

**Pull request merged**